src/main/java/org/moddingx/modlistcreator/changelogger/README.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
```

Comment:
Please move to a `docs` directory in repository root, just as [ModGradle](https://github.com/ModdingX/ModGradle/tree/master/docs) does.